ScreenManager.lua

```diff
@@ -34,6 +34,29 @@ local stack;
 local screens;
 local change;
 
+local allcallbacks = {
```

Comment:
"Constants" in Lua usually are written in allcaps connected by underscores so I'd use `LOVE_CALLBACKS`or `ALL_CALLBACKS`.

The reason why I was asking for a fixed list was that we have better control over the registered functions, but I just saw that you allow the user to pass in a custom table anyway.

If you want to use the original
```
 for name in pairs( love.handlers ) do
     allcallbacks[#allcallbacks + 1] = name
 end
```
I'd thing it's better to move it into the registerCallbacks function because the ALL_CALLBACKS variable isn't needed anywhere else in the class :)